test/wallaby/browser/dialog_test.exs

```diff
@@ -0,0 +1,208 @@
+defmodule Wallaby.Browser.DialogTest do
+  use Wallaby.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "dialogs.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "accept_dialogs/1" do
+    test "accept window.alert", %{page: page} do
+      result = page
+      |> accept_dialogs()
+      |> click(Query.link("Alert"))
+      |> find(Query.css("#result"))
+      |> Element.text()
```

Comment:
hm, this is mostly a style question and dunno how @keathley sees this but I prefer:

```
result = page
         |> thing
         |> other_thing
```

or 

```
result =
  page
  |> thing
  |> other_thing
```

Looks more tidy imo :)